lib/acts_as_list/active_record/acts/sequential_updates_method_definer.rb

```diff
@@ -0,0 +1,21 @@
+module ActiveRecord::Acts::List::SequentialUpdatesDefiner #:nodoc:
```

Comment:
Just realised that this class name doesn't match the file name. Can you change the class name as all the other definer classes are `...MethodDefiner` :)